### ISSUE TITLE
Chokes when there is no metadata

### ISSUE THREAD

**Issue by ArtskydJ** [TehShrike/text-metadata-parser#9]
```js
C:\Users\Michael\Github\KayserCommentaryDeployer\noddity-maker>node
> var tmp = require('text-metadata-parser')
undefined
> tmp('key: value\nkey2: value2\n\nThis is my content')
{ content: 'This is my content',
  metadata: { key: 'value', key2: 'value2' } }
> tmp('This is my content')
{ content: '---\n\n---\nThis is my content', metadata: {} }
> tmp('{{>current}}\n')
{ content: '---\n\n---\n', metadata: {} }
>
```

**Comment by TehShrike** [comment]
Oh yeah, it definitely shouldn't be borking the content with those `---` dashes

**Comment by TehShrike** [comment]
Published in 3.0.3
